recipes/bolt/bld.bat

```diff
@@ -0,0 +1,43 @@
+@echo on
+
+:: until https://github.com/llvm/llvm-project/pull/97130 lands,
+:: follow https://github.com/conda-forge/llvmdev-feedstock/blob/main/recipe/build.sh,
+:: minus the tests and plus LLVM_ENABLE_PROJECTS="bolt" / LLVM_TARGETS_TO_BUILD=...
+
+mkdir build
+cd build
+
+REM remove GL flag for now
+set "CXXFLAGS=-MD"
+set "CC=cl.exe"
+set "CXX=cl.exe"
```

Comment:
Remove these

Reply:
Wasn't the idea to match the llvmdev builds as closely as possible?

Reply:
Done in https://github.com/conda-forge/staged-recipes/pull/22204/commits/004cdc9409acdfe614a1c78582208c1ab564b1a4